cypress/integration/submission-ui.spec.ts

```diff
@@ -1,5 +1,10 @@
-const password = 'admin';
-const email = 'test@test.edu';
+/**
+ * This IT will be never be pushed to the upstream because clicking testing DOM elements is antipattern because
```

Comment:
how is vanilla dspace doing it? No DOM?

Reply:
vanilla dspace is testing DOM elements but it has very simple integration tests - it check only visibility or main simple functions of the DOM elements. Our tests are more difficult.